internal/provider/story_resource.go

```diff
@@ -28,12 +35,37 @@ type storyResourceModel struct {
 	FolderID      types.Int64  `tfsdk:"folder_id"`
 }
 
-type StoryImportRequest struct {
-	NewName  string                 `json:"new_name"`
-	TeamID   int64                  `json:"team_id,omitempty"`
-	FolderID int64                  `json:"folder_id,omitempty"`
-	Mode     string                 `json:"mode,omitempty"`
-	Data     map[string]interface{} `json:"data"`
+type storyResourceModel struct {
+	Data                 types.String `tfsdk:"data"`
+	TinesApiToken        types.String `tfsdk:"tines_api_token"` // Deprecated
+	TenantUrl            types.String `tfsdk:"tenant_url"`      // Deprecated
+	ID                   types.Int64  `tfsdk:"id"`
+	Name                 types.String `tfsdk:"name"`
+	UserID               types.Int64  `tfsdk:"user_id"`
+	Description          types.String `tfsdk:"description"`
+	KeepEventsFor        types.Int64  `tfsdk:"keep_events_for"`
+	Disabled             types.Bool   `tfsdk:"disabled"`
+	Priority             types.Bool   `tfsdk:"priority"`
+	STSEnabled           types.Bool   `tfsdk:"send_to_story_enabled"`
+	STSAccessSource      types.String `tfsdk:"send_to_story_access_source"`
+	STSAccess            types.String `tfsdk:"send_to_story_access"`
+	STSSkillConfirmation types.Bool   `tfsdk:"send_to_story_skill_use_requires_confirmation"`
+	SharedTeamSlugs      types.List   `tfsdk:"shared_team_slugs"`
+	EntryAgentID         types.Int64  `tfsdk:"entry_agent_id"`
+	ExitAgents           types.List   `tfsdk:"exit_agents"`
```

Comment:
This is a little tricky I think right because even from the API this isn't typed and I wonder if that is ok, esp if the return value of Agent could contain non deterministic strings 🤔